### PR TITLE
Tests for createapplication

### LINKED ISSUE #126: Create application use case

### PR DESCRIPTION
**Pull request by cecexyzhang**
It addresses issue #126 

added tests for createapplication and added methods in CreateApplication and CreateApplicationInteractor to support unit tests

### CODE REVIEW

File: src/test/java/usecase/createapplication/CreateApplicationInteractorTest.java
Comment (Raqkkkkk) [inline src/test/java/usecase/createapplication/CreateApplicationInteractorTest.java:1]: Can include the case where the user already has access to the project for full coverage

**Review by Raqkkkkk — COMMENTED** [review]
Overall tests has pretty good coverage and matches with what we expect the use case should do

File: src/test/java/usecase/createapplication/CreateApplicationInteractorTest.java
Comment (cecexyzhang) [reply src/test/java/usecase/createapplication/CreateApplicationInteractorTest.java:1]: Could you add that test case please? Thanks!

**Comment by Raqkkkkk** [comment]
merge since no issues were found

**Pull request merged**